.travis.yml

```diff
@@ -139,7 +139,7 @@ script:
       echo 'image.cmap : viridis' >> $MPL_RC_DIR/matplotlibrc;
       cd $INSTALL_DIR/docs/iris;
       echo `make clean html`;
```

Comment:
This is the real problem; putting the command in backticks means it won't stop the build. Remove `echo `` ` and it should fail.